### PR TITLE
tcp goodput module

### PR DESCRIPTION
**Pull request by hgn**
This merge request extend the TCP goodput module description and enrich the specification with examples. Furthermore, it reduced the tcp-goodput module to a bare minimum. Which is fine for the first iteration.

@monfron if everything is fine **i** will merge it.

### CODE REVIEW

File: mod-tcp-goodput.md [hunk 1]
```diff
@@ -23,25 +28,136 @@ simple data exchange it has some novel features.
...
+
+```
+tcp-goodput
+```
+
+## Messages
+
+### Info Reply
+
+`tcp-goodput` has no additional information for the client. The `tcp-goodput`
```
Comment (monfron) [inline mod-tcp-goodput.md:43]: "E.g. the "tcp-goodput"
   module has a section which specifies allowed values for the tcp-goodput
  module."

contrary statements: 
https://github.com/protocollabs/maparo-std/pull/1/files#r223979114

File: mod-tcp-goodput.md [hunk 2]
```diff
@@ -23,25 +28,136 @@ simple data exchange it has some novel features.
...
+                    "id" = "<uint64_t>",
+                    # port to open and receive data
+                    "port" : "7777",
+                },
+                {
+                    # id, identifying this stream uniquly
+                    "id" = "<uint64_t>",
+                    # port to open and receive data
+                    "port" : "7778",
+                },
```
Comment (monfron) [inline mod-tcp-goodput.md]: snapshot length is missing here...used in maparo pulser...

**Review by monfron — APPROVED** [review]
have a look at my comments

File: mod-tcp-goodput.md (hunk 2 above)
Comment (hgn) [reply mod-tcp-goodput.md]: Right, size should be more appropriate for this. IMHO

File: mod-tcp-goodput.md (hunk 1 above)
Comment (hgn) [reply mod-tcp-goodput.md:43]: additional fields. additional. seq-rq is standard and always required.

**Pull request merged**